plugins/markdown.js

```diff
@@ -1,6 +1,20 @@
 import Vue from 'vue'
-import VueSimpleMarkdown from 'vue-simple-markdown'
-// You need a specific loader for CSS files like https://github.com/webpack/css-loader
-import 'vue-simple-markdown/dist/vue-simple-markdown.css'
+import { VueShowdown } from 'vue-showdown'
+import sanitizeHtml from 'sanitize-html'
 
-Vue.use(VueSimpleMarkdown)
+const NewVueShowdown = {
+    ...VueShowdown,
+    computed: {
+        ...VueShowdown.computed,
+        outputHtml() {
+            if (this.converter) {
+                this.converter.setFlavor(this.flavor || 'allOn')
+                const html = this.converter.makeHtml(this.inputMarkdown)
+                return sanitizeHtml(html)
```

Comment:
My thought here is that we want to also disallow relative `a` `href` attrs.  Can we use the `allowProtocolRelative: false` attr (https://www.npmjs.com/package/sanitize-html#default-options) to accomplish that?

Reply:
Unfortunately no... this filter only filters out links begin with `//`. Here is a (reference)[https://en.wikipedia.org/wiki/Wikipedia:Protocol-relative_URL#:~:text=about%2Fhistory%2F)-,Definition,t%20support%20HTTPS%20at%20all]. We need customized filter to kick out relative hrefs

Reply:
Why do we need to disallow it?

Reply:
On further reflection, I guess there isn't a use case for worrying about relative URLs here.